src/components/quota/QuotaDetails.vue

```diff
@@ -22,6 +22,8 @@ export default {
 
 <style scoped>
 .quota-usage-container {
+  direction: ltr;
```

Comment:
Does it not affected by direction? shouldn't it be calculated? just wandering...

Reply:
@omerzamir  It shows <used> / <limit> and includes the size unit (GB/MB...) and will always be a ltr text.

Reply:
OK. Resolved.